src/test/java/ladder/domain/LineTest.java

```diff
@@ -0,0 +1,44 @@
+package ladder.domain;
+
+import java.util.Collections;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class LineTest {
+    @Test
+    @DisplayName("라인이 비어있으면 예외 발생")
+    void validateEmptyLine() {
+        Assertions.assertThatThrownBy(() -> new Line(Collections.emptyList()))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("적어도 가로 라인이 하나이상 있어야 한다.");
```

Comment:
```suggestion
        List<Stick> emptySticks = Collections.emptyList();
        Assertions.assertThatThrownBy(() -> new Line(emptySticks))
                .isInstanceOf(IllegalArgumentException.class)
                .hasMessage("적어도 가로 라인이 하나이상 있어야 한다.");
```
이 테스트 안에서는 큰 문제 없겠지만, 이왕이면 assert 할 때 검증하고자 하는 변수 또는 메서드 하나만 넣는게 좋아요!